projects/igniteui-angular/src/lib/services/overlay/overlay.ts

```diff
@@ -577,16 +576,6 @@ export class IgxOverlayService implements OnDestroy {
         return info;
     }
 
-    private placeElementHook(element: HTMLElement): HTMLElement {
-        if (!element.parentElement) {
-            return null;
-        }
-
-        const hook = this._document.createElement('div');
```

Comment:
we are using element hook to return the overlay element at its original position. If we do not do this overlay will change the original markup which is not desired.
To fix the issue you can set the `display` of the hook element to `none` like this:
```
hook.style.display = 'none';
```

Reply:
Oh, I see. That was very helpful, I was really wondering what was the reason behind this hook. Will do so ✔